GamerpotFrontend/src/library/constants/tabScreens.ts

```diff
@@ -0,0 +1,43 @@
+import {colors} from './colors';
+import {
+  AccountScreen,
+  DealsScreen,
+  HomeScreen,
+  NewsScreen,
+} from '../../screens';
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+  tabBackground: {
+    backgroundColor: colors.tabBackground,
+    paddingBottom: 4,
+  },
+});
+
```

Comment:
You can export this section to a file for styles.

Reply:
Done